data/presets/traffic_sign/hazard/pedestrians.json

```diff
@@ -0,0 +1,15 @@
+{
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "traffic_sign": "hazard",
+        "hazard": "pedestrian"
+    },
+    "reference": {
+        "key": "hazard",
+        "value": "pedestrian"
+    },
+    "name": "Pedestrian Warning Sign"
```

Comment:
I think this preset would also apply to a sign for pedestrian crossings, so that should be an alias. Most jurisdictions have a sign specifically about pedestrian crossings, whereas recent editions of the MUTCD only have a sign about pedestrians posing a hazard in general.